src/fides/api/util/cache.py

```diff
@@ -200,7 +200,6 @@ def get_cache(should_log: Optional[bool] = False) -> FidesopsRedis:
 
     global _connection  # pylint: disable=W0603
     if _connection is None:
-        logger.debug("Creating new Redis connection...")
```

Comment:
This looks good, but you need to do the same for the `get_read_only_cache` version